### DISCUSSION TITLE
Docker Installation stuck at python-typing

### DISCUSSION THREAD

**Discussion by aarishshahmohsin** [learningequality/kolibri#11367] [Q&A] [answered]
I am trying to locally setup kolibri using docker. I've tried multiple times and every time the installation gets stuck at
 `=> => # Setting up python-typing (3.6.2-1) ... ` .
 Command : ` make docker-build-base`.  
OS: Fedora 38

![Screenshot from 2023-10-07 20-24-47](https://github.com/learningequality/kolibri/assets/49566965/dae538e5-febb-4381-8caf-83351b6fcec4)

**Comment by bjester** [comment]
Hi @aarishshahmohsin,

Thank you for your interest in contributing to Kolibri!

I have not seen this issue before. Could it be a temporary disruption with that particular package?

We've identified several other issues with our current docker-based development setup and are actively working on improving it. So you may unfortunately run into additional issues at this time. Here's another discussion item where other issues have been observed: https://github.com/learningequality/kolibri/discussions/11353

**Comment by aarishshahmohsin** [comment]
I was able to fix the issue by modifying the base.dockerfile to install the packages with a different method. I think the issue was because of some packages being deprecated .

**Reply by bjester** [reply, marked as answer]
Okay, thanks for following up. We have some [imminent updates](https://github.com/learningequality/kolibri/pull/11408) coming to the dockerfile which might help
